src/commands/code-analyzer/rules.ts

```diff
@@ -27,14 +32,22 @@ export default class RulesCommand extends SfCommand<void> {
 		view: Flags.string({
 			summary: getMessage(BundleName.RulesCommand, 'flags.view.summary'),
 			char: 'v',
-			options: ['table', 'detail'] // TODO: Should probably be enum?
+			default: View.TABLE,
+			options: Object.values(View)
 		})
 	};
 
 	public async run(): Promise<void> {
 		const parsedFlags = (await this.parse(RulesCommand)).flags;
+		const uxDisplay: UxDisplay = new UxDisplay(this);
+		const dependencies: RulesDependencies = {
+			configLoader: new ConfigLoaderImpl(),
+			engineLoader: new EngineLoaderImpl(),
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison -- We know `parsedFlags.view` is a valid value.
```

Comment:
The linter complains if you do a comparison of a string literal to an enum value, because usually you can't guarantee that the string is a valid enum option. Due to circumstances, we actually _can_ guarantee that, so I'm silencing the rule.

Reply:
Would it help to create a isView() utility method that checks parsedFlags.view is part of the View values before calling this comparison?

Reply:
I'm not actually sure. I can futz with it a little to see if I can find a way around this.